yarn-project/cli/src/index.ts

```diff
@@ -278,21 +283,52 @@ export function getProgram(log: LogFn, debugLogger: DebugLogger): Command {
   program
     .command('get-logs')
     .description('Gets all the unencrypted logs from L2 blocks in the range specified.')
-    .option('-f, --from <blockNum>', 'Initial block number for getting logs (defaults to 1).')
-    .option('-l, --limit <blockCount>', 'How many blocks to fetch (defaults to 100).')
+    .option('-tx, --txHash <txHash>', 'A transaction hash to get the receipt for.', parseOptionalTxHash)
+    .option(
+      '-f, --fromBlock <blockNum>',
+      'Initial block number for getting logs (defaults to 1).',
+      parseOptionalInteger,
+    )
+    .option('-t, --toBlock <blockNum>', 'Up to which block to fetch logs (defaults to latest).', parseOptionalInteger)
+    .option('-c, --contractAddress <contractAddress>', 'Contract address to filter logs by.', parseOptionalAztecAddress)
```

Comment:
We're using `-ca, --contract-address` in other parts of the CLI. Though we could probably just name it `address`, since all addresses refer to a contract, right? But that's a bigger discussion.

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/2600/commits/0301f09aa9084a3b94efdc9339ed7fe0430fbbb4 .

> Though we could probably just name it address, since all addresses refer to a contract, right? But that's a bigger discussion.
Makes sense but we would have to educate devs. I can imagine devs being confused when not knowing this and seeing just an address.